custom_components/pyscript/trigger.py

```diff
@@ -812,6 +822,41 @@ async def trigger_watch(self):
                 if self.task_unique is not None:
                     task_unique_func = Function.task_unique_factory(action_ast_ctx)
 
+                #
+                # check for changes to state or attributes
+                #
+
+                # if "value" not in func_args, then we are state_check_now
+                if "value" in func_args and notify_type == 'state':
+                    trig_ident_change = False
+
```

Comment:
Might be easier to extract `var_name`, `value`, `old_value` as variables, just to reduce the repetitive code below.